### PR TITLE
test(perf_simple_query): x86_64 job

### PR DESCRIPTION
**Pull request by temichus**
Run perf_simple_query on  x86_64

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by temichus** [comment]
now Reports are collected properly but without any mention of node type. this will be added in the PR  https://github.com/scylladb/scylla-cluster-tests/pull/6221

**Review by aleksbykov — DISMISSED** [review]
LGTM, small improvements

File: test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+cluster_backend: 'aws'
+
+n_loaders: 0
+n_monitor_nodes: 0
+n_db_nodes: 1
+use_mgmt: false
+
+instance_type_db: 'i4i.2xlarge'
+data_volume_disk_num: 1
+user_prefix: 'perf-microbenchmarking-amazon'
```
[the remark this answers is not in the document]
Comment (temichus) [reply test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml]: fixed

File: test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+test_duration: 60
+backtrace_decoding: false
+cluster_backend: 'aws'
+
+n_loaders: 0
+n_monitor_nodes: 0
+n_db_nodes: 1
+use_mgmt: false
+
+instance_type_db: 'i4i.2xlarge'
```
Comment (roydahan) [inline test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml:10]: Why i4i is chosen?
Why not the equivalent of the one that was chosen to run for arm?

File: test-cases/microbenchmarking/amazon_perf_simple_query_c7g.yaml [hunk 3]
```diff
@@ -9,7 +9,7 @@ use_mgmt: false
 
 instance_type_db: 'c7g.large'
 data_volume_disk_num: 1
-user_prefix: 'perf-microbenchmarking-amazon'
+user_prefix: 'perf-microbenchmarking-amazon-c7g-large'
```
Comment (roydahan) [inline test-cases/microbenchmarking/amazon_perf_simple_query_c7g.yaml:12]: I don't think we should have this in the name, no one understands it. just ARM and x86 in the name.

File: jenkins-pipelines/perf-simple-query-i4i.jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    test_name: 'microbenchmarking_test.PerfSimpleQueryTest.test_perf_simple_query',
+    test_config: 'test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml',
```
Comment (roydahan) [inline jenkins-pipelines/perf-simple-query-i4i.jenkinsfile:10]: We don't need this entire yaml duplicated, only a configuration yaml with one parameter that was changed.

**Review by roydahan — CHANGES_REQUESTED** [review]
Please see my comments.

File: test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml (hunk 2 above)
Comment (temichus) [reply test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml:10]: agree will set c6i.large

File: test-cases/microbenchmarking/amazon_perf_simple_query_c7g.yaml (hunk 3 above)
Comment (temichus) [reply test-cases/microbenchmarking/amazon_perf_simple_query_c7g.yaml]: renamed

File: jenkins-pipelines/perf-simple-query-i4i.jenkinsfile (hunk 4 above)
Comment (temichus) [reply jenkins-pipelines/perf-simple-query-i4i.jenkinsfile]: changed, have to change [sdcm/sct_config.py](https://github.com/scylladb/scylla-cluster-tests/pull/6512/files#diff-4964b8575600051f35ba15bc41a6cdd5576162b0882a61eea12b3013091c67ab) little as well

File: test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml (hunk 2 above)
Comment (temichus) [reply test-cases/microbenchmarking/amazon_perf_simple_query_i4i.yaml]: fixed

File: sdcm/sct_config.py [hunk 5]
```diff
@@ -2018,8 +2018,8 @@ def verify_configuration(self):
 
         self._verify_data_volume_configuration(backend)
 
-        self._validate_seeds_number()
         if self.get('n_db_nodes'):
+            self._validate_seeds_number()
```
Comment (roydahan) [inline sdcm/sct_config.py:2022]: I don't understand how this change is related and probably it should be in a different commit or explain in the commit message.

Comment (temichus) [reply sdcm/sct_config.py:2022]: linting test error:
```

[2023-08-21T09:58:05.833Z] ---- linting: ./test-cases/microbenchmarking/amazon_perf_simple_query_x86.yaml -----

[2023-08-21T09:58:05.833Z] Traceback (most recent call last):

[2023-08-21T09:58:05.833Z]   File "/tmp/jenkins/workspace/can_scylla-cluster-tests_PR-6512/./sct.py", line 766, in _run_yaml_test

[2023-08-21T09:58:05.833Z]     config.verify_configuration()

[2023-08-21T09:58:05.833Z]   File "/tmp/jenkins/workspace/can_scylla-cluster-tests_PR-6512/sdcm/sct_config.py", line 2021, in verify_configuration

[2023-08-21T09:58:05.833Z]     self._validate_seeds_number()

[2023-08-21T09:58:05.833Z]   File "/tmp/jenkins/workspace/can_scylla-cluster-tests_PR-6512/sdcm/sct_config.py", line 2092, in _validate_seeds_number

[2023-08-21T09:58:05.833Z]     num_of_db_nodes = sum([int(i) for i in str(self.get('n_db_nodes')).split(' ')])

[2023-08-21T09:58:05.833Z]   File "/tmp/jenkins/workspace/can_scylla-cluster-tests_PR-6512/sdcm/sct_config.py", line 2092, in <listcomp>

[2023-08-21T09:58:05.833Z]     num_of_db_nodes = sum([int(i) for i in str(self.get('n_db_nodes')).split(' ')])

[2023-08-21T09:58:05.833Z] ValueError: invalid literal for int() with base 10: 'None'
```


due to config file have no 'n_db_nodes' but function  self._validate_seeds_number() is still require it:
```
 def _validate_seeds_number(self):
        seeds_num = self.get('seeds_num')
        assert seeds_num > 0, "Seed number should be at least one"

        num_of_db_nodes = sum([int(i) for i in str(self.get('n_db_nodes')).split(' ')])
        assert not num_of_db_nodes or seeds_num <= num_of_db_nodes, \
            f"Seeds number ({seeds_num}) should be not more then nodes number ({num_of_db_nodes})"
```

Comment (temichus) [reply sdcm/sct_config.py:2022]: commit message has updated

**Review by roydahan — APPROVED** [review]
